### PR TITLE
Wiki Updation #2

### PR DESCRIPTION
**Pull request by Priyanuj-Gogoi**

### CODE REVIEW

**Comment by MineBartekSA** [comment]
Oh boy, 57 files.... this will take me a while to review

File: src/bdscript/footerIcon.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ $footerIcon[imageURL;(optional) embedIndex]
 
 ### Breakdown
 - `imageURL` - The URL to set the footer icon as. Must be a valid image URL.
-- `embedIndex` - What embed the footer icon should belong to. Default is `1`. [(learn more)](https://nilpointer-software.github.io/bdfd-wiki/guides/embedIndexes.html)
+- `embedIndex` - What embed the footer icon should belong to. Default is `1`. [(learn more)](/src/resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/footerIcon.md:12]: Please correct arguments
[JSON function data](https://botdesignerdiscord.com/public/api/function/$footerIcon[])

File: src/bdscript/image.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ $image[imageURL;(optional) embedIndex]
 
 ### Breakdown
 - `imageURL` - The URL of the image that appears. Must be a valid image URL.
-- `embedIndex` - What embed the image should belong to. Default is `1`. [(learn more)](https://nilpointer-software.github.io/bdfd-wiki/guides/embedIndexes.html)
+- `embedIndex` - What embed the image should belong to. Default is `1`. [(learn more)](/src/resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/image.md:11]: Please correct arguments
[JSON function data](https://botdesignerdiscord.com/public/api/function/$image[])

File: src/bdscript/lowestRoleWithPerms.md [hunk 3]
```diff
@@ -3,11 +3,11 @@ Returns the lowest role in the server that has all the provided permissions.
 
 ## Usage
 ```
-$lowestRoleWithPerms[permissions]
+$lowestRoleWithPerms[permissions;...]
 ```
 
 ### Breakdown
-- `permissions` - The permissions that the role needs to have.  [(see permissions list)](https://nilpointer-software.github.io/bdfd-wiki/guides/permissions.html)
+- `permissions` - The permissions that the role needs to have.  [(see permissions list)](/src/resources/embedIndexes.md). Use `;` as separator for multiple permissions
```
Comment (MineBartekSA) [inline src/bdscript/lowestRoleWithPerms.md:10]: Wrong link destination

File: src/bdscript/onlyBotChannelPerms.md [hunk 4]
```diff
@@ -8,5 +8,5 @@ $onlyBotChannelPerms[channelID;permissions;errorMessage]
 
 ### Breakdown
 - `channelID` - The channel to check the permissions for. Use `$channelID` for the current channel.
-- `permissions` - The [permissions](https://nilpointer-software.github.io/bdfd-wiki/guides/permissions.html) that the bot needs to execute the command. Separate permissions with `;`.
+- `permissions` - The [permissions](/src/resources/permissions.md) that the bot needs to execute the command. Separate permissions with `;`.
```
Comment (MineBartekSA) [inline src/bdscript/onlyBotChannelPerms.md:11]: Please correct arguments in the full function
[JSON function data](https://botdesignerdiscord.com/public/api/function/$onlyBotChannelPerms[])

File: src/bdscript/onlyBotPerms.md [hunk 5]
```diff
@@ -7,5 +7,5 @@ $onlyBotPerms[permissions;errorMessage]
 ```
 
 ### Breakdown
-- `permissions` - The permissions that the bot needs to run the command.
+- `permissions` - The [permissions](/src/resources/permissions.md) that the bot needs to run the command.
```
Comment (MineBartekSA) [inline src/bdscript/onlyBotPerms.md:10]: Please correct arguments in the full function
[JSON function data](https://botdesignerdiscord.com/public/api/function/$onlyBotPerms[])

File: src/bdscript/removeSplitTextElement.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # $removeSplitTextElement
 Removes a certain element from the [$textSplit](https://nilpointer-software.github.io/bdfd-wiki/bdscript/textSplit.html) values.
```
Comment (MineBartekSA) [inline src/bdscript/removeSplitTextElement.md:2]: Please correct the link destination to be relative

File: src/bdscript/thumbnail.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ $thumbnail[imageURL;(optional) embedIndex]
 
 ### Breakdown
 - `imageURL` - The image to set as the thumbnail.
-- `embedIndex` - What embed the thumbnail should belong to. Default is `1`. [(learn more)](https://nilpointer-software.github.io/bdfd-wiki/guides/embedIndexes.html)
+- `embedIndex` - What embed the thumbnail should belong to. Default is `1`. [(learn more)](../resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/thumbnail.md:11]: Please correct arguments
[JSON function data](https://botdesignerdiscord.com/public/api/function/$thumbnail[])

File: src/bdscript/title.md [hunk 8]
```diff
@@ -7,8 +7,8 @@ $title[text;(optional) embedIndex]
 ```
 
 ### Breakdown
-- `text` - The text to set the title as.
-- `embedIndex` - What embed the title should belong to. Default is `1`. [(learn more)](https://nilpointer-software.github.io/bdfd-wiki/guides/embedIndexes.html)
+- `text` - The text to set the title as. It cannot exceed more than 256 characters.
+- `embedIndex` - What embed the title should belong to. Default is `1`. [(learn more)](../resources/embedIndexes.md)
```
Comment (MineBartekSA) [inline src/bdscript/title.md:11]: Please correct arguments (`text` can stay)
[JSON function data](https://botdesignerdiscord.com/public/api/function/$title[])

**Comment by ShiroNotFriday** [comment]
> Oh boy, 57 files.... this will take me a while to review

Just in case, I prepared a coffin for you. 👌

**Review by MineBartekSA — CHANGES_REQUESTED** [review]
Thanks for taking your time to improve our wiki.
But there are a few changes I'd like you to make.
Also, please correct arguments in `$authorURL[]`

File: src/bdscript/editEmbedIn.md [hunk 9]
```diff
@@ -1,18 +1,19 @@
...
 
 ### Breakdown
-- `time` - The time to wait before editing the message (e.g `3s`, `30s`, `10m`). Max is `40m`. Required.
+- `time` - The time to wait before editing the message (e.g `3s`, `30s`, `10m`). Max time is 40 minutes (for premium users, it's 120 minutes). Required.
+
 - `title` - The new embed title. Optional.
 - `description` - The new embed description. Optional.
 - `footer` - The new embed footer. Optional.
-- `color` - The embed border color, must be a valid [color hex](https://htmlcolorcodes.com/color-picker). Required (use `0` for default color).
+- `color` - The embed border color, must be a valid [color hex](https://htmlcolorcodes.com/color-picker) or color integer number. Use `0` for default color.
```
Comment (MineBartekSA) [inline src/bdscript/editEmbedIn.md:16]: The `color` argument **is** optional, so please add this information to its description

File: src/bdscript/editEmbedIn.md [hunk 10]
```diff
@@ -1,18 +1,19 @@
 # $editEmbedIn
-Edits the bot's message after the given time, as a embed.
+Edits the bot's message after the given time, as an embed.
 
 ## Usage
 ```
 $editEmbedIn[time;title;description;footer;color]
```
Comment (MineBartekSA) [inline src/bdscript/editEmbedIn.md:6]: Please add round brackets
[JSON function data](https://botdesignerdiscord.com/public/api/function/$editEmbedIn[])

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thanks for your hard work

**Pull request merged**